### DISCUSSION TITLE
Welcome to Data4Policy Discussions!

### DISCUSSION THREAD

**Discussion by parvathykrishnank** [undpindia/dicra#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by archit1990** [comment]
request to get the access to API data set for analyzing data set for climate prediction model

**Reply by archit1990** [reply]
email id acceleratorlab.in@undp.org is not working
